jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -494,7 +494,7 @@
             </glue-state-sync-wrapper>
           </div>
         </div>
-
+      </div>

```

Comment:
good catch - it does look like there is an unclosed `div`, but I _think_ it needs to be closed after the spinner div instead of before.

Reply:
Thanks – I had a really hard time tracking down where it actually belonged. A linter gave me the tip to put it there, but I wasn't convinced.